File: lab2/Human.cpp
#include "Human.h"

void Human::SetName(std::string name)
{
	this->name = name;
}

void Human::SetLastname(std::string lastname)
{
	this->lastname = lastname;
}

void Human::SetAge(int age)
{
	this->age = age;
}

void Human::GetInfo()
{
	std::cout << lastname << " " << name << " " << age << std::endl;
}

std::string Human::GetLastname()
{
	return lastname;
}

File: lab3/game/development/Human.java
package game.development;

public abstract class Human extends Contact{
    private String name;
    private String lastname;
    private byte age;
    private String skill;

    public void setPersonalInfo(String name,String lastname,byte age,String skill) {
        this.name = name;
        this.lastname = lastname;
        this.age = age;
        this.skill = skill;
    }
    public String getPersonalInfo() {
        String info = "";
        info += lastname;
        info += name;
        info += age;
        info += "Skills: " + skill;
        return info;
    }
}
